README.md

```diff
@@ -122,6 +122,8 @@ Build:
 
 `npm run dist`
 
-To copy UI Kit into an existing project:
+To copy UI Kit into an existing project, run the following:
 
-`cp -r dist/ ../project/node_modules/@dcos/ui-kit/`
+`cp -r dist/ ./<project>/node_modules/@dcos/ui-kit/`
+
+Replace `<project>` with the associated host project name and folder structure.
```

Comment:
Should we also say that they'll need to restart the host project app? It might be obvious to some, but not others (i.e. people like me who need explicit instructions for everything)

Reply:
Pushed up an update to clarify that, let me know what you think.